Packs/UrlScan/Integrations/UrlScan/UrlScan.yml

```
@@ -142,6 +147,8 @@ script:
       defaultValue: 'false'
       required: false
       secret: false
+    - name: useragent
+      description: User agent to perform request
```

```suggestion
    - default: false
      isArray: false
      name: useragent
      description: User agent to perform request
      required: false
      secret: false
```
I added it just to keep the structure of the existing arguments